api/v1/ytsaurus_types.go

```diff
@@ -779,15 +782,18 @@ const (
 	UpdateSelectorEverything UpdateSelector = "Everything"
 )
 
-type UpdateFlow string
+type ComponentUpdateSelector struct {
+	//+optional
+	ComponentType consts.ComponentType `json:"componentType,omitempty"`
+	//+optional
+	ComponentGroup consts.ComponentGroup `json:"componentGroup,omitempty"`
+	//+optional
+	ComponentName string `json:"componentName,omitempty"`
```

Comment:
Maybe if we are in context of components let's just have `Type, Name, Group` fields?